core/config/locales/en.yml

```diff
@@ -730,6 +730,7 @@ en:
     charged: Charged
     checkout: Checkout
     checkout_message: Checkout message
+    checkout_settings: Checkout settings
```

Comment:
@mad-eel let's use a combination of `Spree.t(:settings) + Spree.t(:checkout)` and not add a new key